interview-questions.md

```diff
@@ -4,39 +4,39 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer:you would use it to call the base classe's constructor
```

Comment:
Just about! You totally have the right idea here, but we would use it to call the parent class's constructor and methods into the child that super is being used without having to rewrite most of the code.